pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Producer.java

```diff
@@ -298,7 +344,8 @@ public void completed(Exception exception, long ledgerId, long entryId) {
                     if (!(exception instanceof TopicClosedException)) {
                         // For TopicClosed exception there's no need to send explicit error, since the client was
                         // already notified
-                        producer.cnx.ctx().writeAndFlush(Commands.newSendError(producer.producerId, sequenceId,
+                        long callBackSequenceId = lastSequenceId >= sequenceId ? lastSequenceId : sequenceId;
```

Comment:
can this be simplified with the following statement?

```
long callbackSequenceId = Math.max(lastSequenceId, sequenceId);
```

Reply:
I will add a common function to get the callback sequenceId